web/client/utils/FeatureTypeUtils.js

```diff
@@ -25,7 +25,10 @@ const types = {
     // 'xsd:gMonthDay': 'date',
     // 'xsd:gYear': 'date',
     // 'xsd:gYearMonth': 'date',
-    // 'xsd:time': 'date',
+    // date-time
+    'xsd:date-time': 'date-time',
```

Comment:
[xsd:dateTime](http://books.xmlschemata.org/relaxng/ch19-77049.html) is the correct format. Please replace 'date-time' everywhere. Make you sure you receive this also in the DescribeFeatureType (check also the camel case).

How did it work?

Reply:
date-time was the type returned from capabilities as you have also checked